plugins/lighthouse/src/components/CreateAudit/index.test.tsx

```diff
@@ -140,7 +140,7 @@ describe('CreateAudit', () => {
 
       await wait(() => expect(rendered.getByLabelText(/URL/)).toBeEnabled());
 
-      expect(useHistory().push).toHaveBeenCalledWith('/lighthouse');
+      expect(useNavigate().push).toHaveBeenCalledWith('/lighthouse');
```

Comment:
is this supposed to still be push?

Reply:
Nope, I just haven’t got round to fixing the tests yet haha. This was find and replace 😂